README.md

```diff
@@ -45,6 +45,8 @@ Fontra Pak should also work on Linux and other platforms, but you will have to b
 
   `npm install`
 
+  `npm run jsdeps`
```

Comment:
With the build hook in place, is this even needed?

Reply:
Instead, we need to mention that Node needs to be installed, probably right after "Ensure you have Python >= 3.10 installed"

Reply:
I already removed it, I think you looked an old version of the workflow.

Reply:
I update the README.

Reply:
(My comments are indeed about the README)

Reply:
Ah ok, sorry again :)